external-crates/move/crates/move-binary-format/src/unit_tests/signature_token_tests.rs

```diff
@@ -49,7 +49,7 @@ fn serialize_nested_types_too_deep() {
 fn deserialize_struct_inst_arity_0() {
     let cursor = Cursor::new(
         [
-            SerializedType::STRUCT_INST as u8,
+            SerializedType::DATATYPE_INST as u8,
             0x0, /* struct handle idx */
```

Comment:
This actually is the exact same for enums since this is deserializing a datatype instantiation, which is the same for both struct and enum types. But also, I realized these tests should really be renamed with `s/struct/datatype` so will do that.